packages/@react-spectrum/codemods/package.json

```diff
@@ -23,6 +22,7 @@
     "@babel/parser": "^7.24.5",
     "@babel/traverse": "^7.24.5",
     "@babel/types": "^7.24.5",
+    "@react-spectrum/s2": "workspace:^",
```

Comment:
not ready for this syntax yet, just use '1.0.0' or whatever version this package is at